litani

```diff
@@ -664,15 +668,15 @@ async def run_build(args):
         for build in builds:
             logging.debug(build)
             ninja.build(**build)
-
+    draw_dependency_graph = False if args.no_draw_dependency_graph else True
     report_dir = litani.get_report_dir()
     run = litani_report.get_run_data(cache_dir)
     lib.validation.validate_run(run)
-    litani_report.render(run, report_dir)
+    litani_report.render(run, report_dir, draw_dependency_graph)
```

Comment:
`, not args.no_draw_dependency_graph`

Reply:
Furthermore, if you finish #55 , you'll be able to pass this variable just once to a `PipelineDepgraphRenderer` and then pass the `PipelineDepgraphRenderer` object to all these calls